plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BooleanWriteFunction.java

```diff
@@ -27,4 +29,30 @@ default Class<?> getJavaType()
 
     void set(PreparedStatement statement, int index, boolean value)
             throws SQLException;
+
+    static BooleanWriteFunction of(int nullJdbcType, BooleanWriteFunctionImplementation implementation)
+    {
+        requireNonNull(implementation, "implementation is null");
+
+        return new BooleanWriteFunction() {
+            @Override
+            public void set(PreparedStatement statement, int index, boolean value) throws SQLException
```

Comment:
nit: should `throws SQLException` be on a newline, or does that not hold for anonymous classes?